### ISSUE TITLE
Not showing the name of a brand new character

### ISSUE THREAD

**Issue by x1marT** [ESX-PUBLIC/esx_scoreboard#5]
When you join a server for the first time and create your first character esx_scoreboard does not read the character name, you need to log out and log in again to refresh it

**Comment by HumanTree92** [comment]
Yeah what you could do is say every 5mins it will refresh it. Only like 5lines of code.

**Comment by HawaiiBeach** [comment]
Easy to fix, I'll make so that esx_scoreboard listens to a `esx_identity:characterUpdated` event

**Comment by CruelAlpha** [comment]
Waiting on implementation 👍

**Comment by HawaiiBeach** [comment]
Its better if people implemented this themselves as it will be different depending on your code.

**Comment by ScaredDonut** [comment]
Seems to only show my OOC name regardless if I relog or not. I tried to restart the server as well. Might be doing something wrong

**Comment by MrSebas2023** [comment]
So, how would I show the current character name instead of steam name?

**Comment by JoanEkaleo** [comment]
Does anyone know how to change from steam name to a Character name? with firtname and last name? been trying to change it.

**Comment by ScaredDonut** [comment]
Make a function that changes a players name to their RP name based on their identifier. That _SHOULD_ work.

**Comment by JoanEkaleo** [comment]
Do u have the example of the function?
been trying to find it.

**Comment by ScaredDonut** [comment]
make a function that checks for identifier. Then do a select statement that matches against the identifier and the returns firstname and lastname.
